docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -159,6 +159,20 @@ See the https://playwright.dev/docs/api/class-page#page-get-attribute[`page.getA
 <2> Use the assertion library provided by the Synthetics agent to look for the
 expected value. See https://jestjs.io/docs/expect[Jest expect docs] for more information.
 
+[discrete]
+[[synthetics-create-test-script-recorder]]
+=== Synthetics Recorder
+
+beta[]
+
+Use the Synthetics Recorder to generate code for one or more steps by interacting with a web page directly.
+
+// In what circumstances should you use Synthetics Recorder?
+// Some high-level details on what it does
```

Comment:
> // Some high-level details on what it does

The recorder launches a special Chromium browser that will listen to the various interactions the user makes with the loaded webpage(s) and record them internally. When the user is ready to export a script, the recorder converts the recorded actions into usable JavaScript code. At that point the user can copy or export the code to a file for use directly with Heartbeat/Synthetics, or upload the code to Kibana for use with the Synthetics Service.